server/services/schedule-service.js

```diff
@@ -10,23 +11,14 @@ module.exports = {
     }
 
     const paymentDates = getPaymentDates()
-    paymentDates.forEach(paymentDate => {
-      scheduleRepository.create({
+
+    for (let i = 0; i < paymentDates.length; i++) {
+      console.log('creating schedule')
+      await scheduleRepository.create({

```

Comment:
this is quite chatty, in future in might be better to replace the 6 database calls with a single bulk insert.

Reply:
Noted.